etna/transforms/decomposition/change_points_based/per_interval_models/base.py

```diff
@@ -10,7 +10,7 @@ class PerIntervalModel(BaseMixin, ABC):
     """Class to handle intervals in change point based transforms.
 
     PerIntervalModel is a class to process intervals between change points
-    in `~etna.transforms.decomposition.change_points_based` transforms.
+    in :py:module:`~etna.transforms.decomposition.change_points_based` transforms.
```

Comment:
Probably, it isn't correct and should be `:py:mod:`

Reply:
It's just about rendering. If it renders correctly, what is the question?